airflow/www/templates/airflow/dag.html

```diff
@@ -26,24 +26,52 @@
 
 {% block content %}
 <div>
-    <h3 class="pull-left">
-      {% if dag.parent_dag is defined and dag.parent_dag %}
-        <span style='color:#AAA;'>SUBDAG: </span> <span> {{ dag.dag_id }}</span>
-      {% else %}
-        <input id="pause_resume" dag_id="{{ dag.dag_id }}" type="checkbox" {{ "checked" if not dag.is_paused else "" }} data-toggle="toggle" data-size="mini" method="post">
-        <span style='color:#AAA;'>DAG: </span> <span> {{ dag.dag_id }}</span> <small class="text-muted"> {{ dag.description }} </small>
-      {% endif %}
-      {% if root %}
-        <span style='color:#AAA;'>ROOT: </span> <span> {{ root }}</span>
-      {% endif %}
-    </h3>
-    <h4 class="pull-right">
-      <a class="label label-default" href="{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}">
-        schedule: {{ dag.schedule_interval }}
-      </a>
-    </h4>
+  <div class="clearfix">
+    <div class="pull-right">
+      <div class="btn-group">
+        <a href="{{ url_for('Airflow.trigger', dag_id=dag.dag_id, origin=request.base_url ) }}"
+           onclick="return confirmTriggerDag('{{ dag.safe_dag_id }}')"
+           class="btn btn-primary"
+        >
+          <span class="glyphicon glyphicon-play-circle" aria-hidden="true"></span>
+          Trigger DAG
+        </a>
+        <a href="{{ url_for('Airflow.refresh', dag_id=dag.dag_id) }}"
+           class="btn btn-primary"
+        >
+          <span class="glyphicon glyphicon-refresh" aria-hidden="true"></span>
+          Refresh
+        </a>
+        <a href="{{ url_for('Airflow.delete', dag_id=dag.dag_id) }}"
+           onclick="return confirmDeleteDag('{{ dag.safe_dag_id }}')"
+           class="btn btn-danger"
+        >
+          <span class="glyphicon glyphicon-remove-circle" aria-hidden="true"></span>
+          Delete
+        </a>
+      </div>
+    </div>
+    <div style="display: flex; align-items: center;">
+      <h2>
+        {% if dag.parent_dag is defined and dag.parent_dag %}
+          <span style='color:#AAA;'>SUBDAG: </span> <span> {{ dag.dag_id }}</span>
+        {% else %}
+          <input id="pause_resume" dag_id="{{ dag.dag_id }}" type="checkbox" {{ "checked" if not dag.is_paused else "" }} data-toggle="toggle" data-size="mini" method="post">
+          <span style='color:#AAA;'>DAG: </span> <span> {{ dag.dag_id }}</span> <small class="text-muted"> {{ dag.description }} </small>
+        {% endif %}
+        {% if root %}
+          <span style='color:#AAA;'>ROOT: </span> <span> {{ root }}</span>
+        {% endif %}
+      </h2>
+      <h4 style="padding-left: 16px">
```

Comment:
Mnior style point: I'd prefer it if we didn't use inline styles.

Reply:
I would like to soon update Bootstrap to a newer version. Then i will use a bootstrap spacing classes.

See: 
https://getbootstrap.com/docs/4.2/utilities/spacing/

Reply:
I am counting on understanding that this code is enough at the moment. However, this is not a long-term solution.